cmd/scan.go

```diff
@@ -8,14 +8,12 @@ import (
 	"github.com/rebuy-de/aws-nuke/resources"
 )
 
-func Scan(sess *session.Session) <-chan *Item {
+func Scan(region Region) <-chan *Item {
 	items := make(chan *Item, 100)
 
 	go func() {
-		listers := resources.GetListers(sess)
-
-		for _, lister := range listers {
-			r, err := safeLister(lister)
+		for category, lister := range resources.GetListers() {
```

Comment:
This is by now the only place we use the variable name `category`. It is actually resource type name, right? Would love to get rid of `category` once and for all.